extensions/stats/plugin.cc

```diff
@@ -206,15 +205,26 @@ const std::vector<MetricFactory>& PluginRootContext::defaultMetrics() {
           },
           false},
       MetricFactory{"request_bytes", MetricType::Histogram,
-
                     [](const ::Wasm::Common::RequestInfo& request_info)
                         -> uint64_t { return request_info.request_size; },
                     false},
       MetricFactory{"response_bytes", MetricType::Histogram,
-
                     [](const ::Wasm::Common::RequestInfo& request_info)
                         -> uint64_t { return request_info.response_size; },
                     false},
+      // GRPC metrics.
+      MetricFactory{
+          "request_messages", MetricType::Counter,
```

Comment:
should we consider making these `request_messages_total` and `response_messages_total` to align with prom best practices: https://prometheus.io/docs/practices/naming/

Reply:
SG.

Reply:
looks like we went with `_count` instead of `_total` ?

Reply:
Oops, renamed to total.